src/otx/core/data/module.py

```diff
@@ -170,7 +217,18 @@ def train_dataloader(self) -> DataLoader:
                     "sampler": RandomSampler(dataset, num_samples=num_samples),
                 },
             )
-        return DataLoader(**common_args)
+        dataloader: DataLoader = DataLoader(**common_args)
+        if (unlabeled_dataloader := self.unlabeled_dataloader()) is not None:
+            # Utilize the CombinedLoader provided by Lightning to bundle multiple dataloaders.
+            # https://lightning.ai/docs/pytorch/stable/data/iterables.html
+            from lightning.pytorch.utilities import CombinedLoader
+
+            iterables = {
+                "labeled": dataloader,
+                **unlabeled_dataloader,
```

Comment:
Why not just "unlabeled" : unlabeled_dataloader ?

Reply:
In this implementation `unlabeled_dataloader` is a dict. This consists of the same dataset but each dataloader with multiple different transform types. So this dict can have multiple keys.